libs/types/src/lib/service.ts

```diff
@@ -2,8 +2,8 @@ import { MultiLanguageText } from './language';
 
 export interface Service extends ServiceToBeCreated {
   id: string;
+  catalogId: string;
   published: boolean;
-  produces: Output[];
```

Comment:
rebasekrøll?

Reply:
Nei denne gangen var det med vilje 😅 Flytter produces ned til serviceToBeCreated isteden for service (som extender serviceToBeCreated)